src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -74,7 +73,18 @@ void AbstractTableGenerator::generate_and_store() {
   }
 
   /**
-   * Encode the Tables
+   * Finalizing all chunks of all tables
```

Comment:
In my opinion we don't need output and time measurements for finalizing, what do you think?